plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -142,5 +197,12 @@ def container
         containers.first
       end
     end
+
+    def sidecar_container
+      @sidecar ||= begin
+        containers = template.spec.template.try(:spec).try(:containers) || []
```

Comment:
maybe just blow up then ... but I guess either way is fine ... will just
blow up somewhere else then ...

On Wed, May 25, 2016 at 10:13 AM, ian waters <notifications@github.com>
wrote:

> In plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb
> <https://github.com/zendesk/samson/pull/1019#discussion_r64612192>:
>
> > @@ -142,5 +197,12 @@ def container
> >          containers.first
> >        end
> >      end
> > +
> > +    def sidecar_container
> > +      @sidecar ||= begin
> > +        containers = template.spec.template.try(:spec).try(:containers) || []
>
> I don't think a template w/o a spec is valid.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/1019/files/c39818e2154e6c5eee4b0917d3b11b4cd0ea9da0#r64612192>
>